tests/_async/test_client_with_auto_confirm_enabled.py

```diff
@@ -0,0 +1,358 @@
+
+
+@pytest.mark.asyncio
+@pytest.mark.depends(on=[test_sign_in.__name__])
+async def test_get_user(client: AsyncGoTrueClient):
+    try:
+        await client.init_recover()
+        response = client.user()
+        assert isinstance(response, User)
+        assert response.id
+        assert response.email == email
+        assert response.email_confirmed_at
+        assert response.last_sign_in_at
+        assert response.created_at
+        assert response.updated_at
+        assert response.app_metadata
+        provider = response.app_metadata.get("provider")
+        assert provider == "email"
+    except Exception as e:
+        assert False, str(e)
+
+
+@pytest.mark.asyncio
+@pytest.mark.depends(on=[test_sign_in.__name__])
+async def test_update_user(client: AsyncGoTrueClient):
+    try:
+        await client.init_recover()
+        response = await client.update(
+            attributes=UserAttributes(data={"hello": "world"})
+        )
+        assert isinstance(response, User)
+        assert response.id
+        assert response.email == email
+        assert response.email_confirmed_at
+        assert response.last_sign_in_at
+        assert response.created_at
+        assert response.updated_at
+        assert response.user_metadata
+        assert response.user_metadata.get("hello") == "world"
+    except Exception as e:
+        assert False, str(e)
+
+
+@pytest.mark.asyncio
+@pytest.mark.depends(on=[test_update_user.__name__])
+async def test_get_user_after_update(client: AsyncGoTrueClient):
+    try:
+        await client.init_recover()
+        response = client.user()
+        assert isinstance(response, User)
+        assert response.id
+        assert response.email == email
+        assert response.email_confirmed_at
+        assert response.last_sign_in_at
+        assert response.created_at
+        assert response.updated_at
+        assert response.user_metadata
+        assert response.user_metadata.get("hello") == "world"
+    except Exception as e:
+        assert False, str(e)
+
+
+@pytest.mark.asyncio
+@pytest.mark.depends(on=[test_get_user_after_update.__name__])
+async def test_sign_out(client: AsyncGoTrueClient):
+    try:
+        await client.sign_out()
+    except Exception as e:
+        assert False, str(e)
+
+
+@pytest.mark.asyncio
+@pytest.mark.depends(on=[test_sign_out.__name__])
+async def test_get_user_after_sign_out(client: AsyncGoTrueClient):
+    try:
+        await client.init_recover()
+        response = client.user()
+        assert not response
+    except Exception as e:
+        assert False, str(e)
+
+
+@pytest.mark.asyncio
+@pytest.mark.depends(on=[test_get_user_after_sign_out.__name__])
+async def test_sign_in_with_the_wrong_password(client: AsyncGoTrueClient):
+    try:
+        await client.sign_in(email=email, password=password + "2")
+        assert False
+    except ApiError:
+        assert True
+    except Exception as e:
+        assert False, str(e)
+
+
+# @pytest.mark.asyncio
```

Comment:
Were these comments meant to be left inside?

Reply:
Oh my bad, that's from test_client_with_auto_confirm_disabled.py that I copy and paste and I forgot to remove it 😅